forTwinCatRepos/.pre-commit-config.yaml

```
@@ -9,8 +9,9 @@ repos:
         files: \.(TcPOU|TcDUT|TcGVL)$
 
 -   repo: https://github.com/pcdshub/pre-commit-hooks.git
-    rev: v1.0.0
+    rev: v1.1.0
```

Let's pre-emptively bump this one more so it matches the tag I'll make after we merge this
```suggestion
    rev: v1.2.0
```